lib/github-glue.ts

```diff
@@ -108,7 +108,7 @@ export class GitHubGlue {
 
         const url = GitGitGadget.parsePullRequestURL(pullRequestURL);
         const pr = await this.getPRInfo(url[0], url[2]);
-        const ccNow = pr.body.match(/\r?\n\r?\n(cc:.*)/is);
+        const ccNow = pr.body.match(/\r?\n\s*\r?\n(cc:.*)/is);
```

Comment:
Wouldn't this also match something like

```
In this patch series, we ensure that `git send-email` adds a
Cc: header whenever encountering even unknown footers,
as long as they are in the format "Some Name
<some@email.com>"
```

?

Reply:
I was kind of hoping to avoid fully parsing the PR body.  In the referenced git PR, [this](https://github.com/gitgitgadget/gitgitgadget/blob/841ad8c6d483e0244fe0cd03c73778775c8e2c61/lib/patch-series.ts#L288) only works because there is a second paragraph in the body, causing much of the body to be treated as a footer (but added back).  `parsePullRequestBody` would also get hit by a line starting with `cc:` like your sample (not sure if the `Cc:` is really at the start of a line).

This change does not change the problem of unknown footers containing a matching email, causing the `cc:` to **not** be added.  I am contemplating a fuller fix so please just ignore for now.  I did notice a couple of matches that need to be made case insensitive.